autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -101,13 +100,8 @@ def _predict_proba(self, X: np.ndarray) -> np.ndarray:
         loader = self.named_steps['data_loader'].get_loader(X=X)
         pred = self.named_steps['network'].predict(loader)
         if isinstance(self.dataset_properties['output_shape'], int):
-            proba = pred[:, :self.dataset_properties['output_shape']]
-            normalizer = proba.sum(axis=1)[:, np.newaxis]
-            normalizer[normalizer == 0.0] = 1.0
-            proba /= normalizer
-
-            return proba
-
+            # The final layer is always softmax now (`pred` already gives pseudo proba)
+            return pred
         else:
```

Comment:
We do not change this else statement in this PR or did you forget?

Reply:
why dont we need this else? I know it was not there in the reg cocktails but here we are raising an error for the user to notify that we don't support other output shapes. I think its fine or even necessary